src/worker.cc

```diff
@@ -1414,6 +1470,13 @@ void ParseApi(const http_request_t& request, valhalla::Api& api) {
     document.AddMember({kv.first, allocator}, array, allocator);
   }
 
+  if (!document.HasMember("language")) {
```

Comment:
this only checks the json, you need to also check if `options.language` is unset from pbf input

Reply:
actually looks like it short circuits up above so you'll need to maybe regoranize this a bit more to support pbf properly

Reply:
d'oh.... I forgot about the PBF logic short circuiting; it looks like JSON and query string are together though so I *think* it handles those two.